src/website/app/demos/Table/examples/sortableControlled.js

```diff
@@ -28,65 +49,83 @@ be managed by the application as a controlled component via the control prop,
           this.state = initialState;
 
           this.handleSort = this.handleSort.bind(this);
-          this.sortFruitsAscending = this.sortFruitsAscending.bind(this);
-          this.sortFruitsDescending = this.sortFruitsDescending.bind(this);
+          this.sort = this.sort.bind(this);
+          this.sortDairyAscending = this.sortDairyAscending.bind(this);
+          this.sortDairyDescending = this.sortDairyDescending.bind(this);
           this.reset = this.reset.bind(this);
         }
 
         handleSort(sort) {
           this.setState({ sort });
         }
 
-        sortFruitsAscending() {
+        // Easily memoized...
```

Comment:
Are you really sure that you don't want to just memoize this method?  I am introducing `memoize-one` in an upcoming PR anyway.  The description does state that it is just one way to do it.  If not, perhaps we can be more clear that it SHOULD be memoized.

Reply:
Alright, you won me over. 😝

Reply:
On second thought, that's not as trivial of an enhancement as I thought. Saving for future work. I did update the text to "should", though.